### PR TITLE
domain name update

### PR DESCRIPTION
**Pull request by lukehinds**
Update from rekor.dev to sigstore.dev, now that the new domain is live. 

Signed-off-by: Luke Hinds <lhinds@redhat.com>

### CODE REVIEW

**Review by dlorenc — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,18 +2,18 @@
 
 Rekór - Greek for “Record”
 
-Rekor's goals are to provide an immutable tamper resistant ledger of metadata generated within a software projects supply chain.  Rekor will enable software maintainers and build systems to record signed metadata to an immutable record. Other parties can then query said metadata to enable them to make informed decisions on trust and non-repudiation of an object's life-cycle. For more details visit the [rekor website](https://rekor.dev)
+Rekor's goals are to provide an immutable tamper resistant ledger of metadata generated within a software projects supply chain.  Rekor will enable software maintainers and build systems to record signed metadata to an immutable record. Other parties can then query said metadata to enable them to make informed decisions on trust and non-repudiation of an object's life-cycle. For more details visit the [rekor website](https://sigstore.dev)
```
Comment (justaugustus) [inline README.md]:
```suggestion
Rekor's goals are to provide an immutable tamper resistant ledger of metadata generated within a software projects supply chain. Rekor will enable software maintainers and build systems to record signed metadata to an immutable record. Other parties can then query said metadata to enable them to make informed decisions on trust and non-repudiation of an object's lifecycle. For more details visit the [sigstore website](https://sigstore.dev)
```

File: README.md [hunk 2]
```diff
@@ -2,18 +2,18 @@
 
 Rekór - Greek for “Record”
```
Comment (justaugustus) [inline README.md:3]:
```suggestion
```

?

File: README.md [hunk 3]
```diff
@@ -2,18 +2,18 @@
...
-For details on CLI usage, [visit the following link](https://rekor.dev/get_started/client/)
+For details on CLI usage, [visit the following link](https://sigstore.dev/get_started/client/)
 
-If you're interesting in integration with rekor, we have an [OpenAPI swagger editor](https://rekor.dev/swagger/)
+If you're interesting in integration with rekor, we have an [OpenAPI swagger editor](https://sigstore.dev/swagger/)
 
-A public instance of rekor can be found at [api.rekor.dev](https://api.rekor.dev/api/v1/log/)
+A public instance of rekor can be found at [api.sigstore.dev](https://api.sigstore.dev/api/v1/log/)
 
 Rekor allows customized manifests (which term them as types), [type customization is outlined here](https://github.com/sigstore/rekor/tree/main/pkg/types).
```
Comment (justaugustus) [inline README.md:18]:
```suggestion
sigstore allows customized manifests (which term them as types), [type customization is outlined here](https://github.com/sigstore/rekor/tree/main/pkg/types).
```

**Review by justaugustus — CHANGES_REQUESTED** [review]
@lukehinds -- qq: Is the intent to have the tool and repo renamed to `sigstore` as well?

If so, left some suggestions.

File: README.md (hunk 1 above)
Comment (lukehinds) [reply README.md]: Ah good point, so must of those will remain as rekor (as they refer to this project, but the last one is correct and should be changed; `[sigstore website](https://sigstore.dev)`

File: README.md (hunk 3 above)
Comment (lukehinds) [reply README.md:18]: That ones ok as it is, so it's referring to itself (I know, it is confusing!).

**Pull request merged**